src/twisted/internet/interfaces.py

```diff
@@ -1243,14 +1250,14 @@ def seconds():
         """
 
 
-    def callLater(delay, callable, *args, **kw):
+    def callLater(delay, callableObj, *args, **kw):
```

Comment:
If this needs to be changed,  it I think that this needs a separate PR...maybe rename it to `func` as it is the mos popular.... but I think that this needs a separate PR... and a backward compatibiliy warning.

```sh
$ grep -r "def callLater(" src/

src/twisted/protocols/policies.py:    def callLater(self, period, func):
src/twisted/protocols/policies.py:    def callLater(self, period, func):
src/twisted/protocols/policies.py:    def callLater(self, period, func):
src/twisted/test/test_policies.py:    def callLater(self, period, func):
src/twisted/test/test_policies.py:    def callLater(self, period, func):
src/twisted/test/test_policies.py:    def callLater(self, timeout, func, *args, **kwargs):
src/twisted/names/dns.py:    def callLater(self, period, func, *args):
src/twisted/conch/test/test_helper.py:    def callLater(self, timeout, f, *a, **kw):
src/twisted/web/http.py:    def callLater(self):
src/twisted/web/http.py:    def callLater(self, value):
src/twisted/internet/_threadedselect.py:    def callLater(self, *args, **kw):
src/twisted/internet/interfaces.py:    def callLater(delay, callable, *args, **kw):
src/twisted/internet/asyncioreactor.py:    def callLater(self, seconds, f, *args, **kwargs):
src/twisted/internet/cfreactor.py:    def callLater(self, _seconds, _f, *args, **kw):
src/twisted/internet/task.py:    def callLater(self, when, what, *a, **kw):
src/twisted/internet/base.py:    def callLater(self, _seconds, _f, *args, **kw):
src/twisted/internet/_glibbase.py:    def callLater(self, *args, **kwargs):
```

Reply:
I've moved this to: https://github.com/twisted/twisted/pull/1325